composer.json

```diff
@@ -26,5 +28,27 @@
         "psr-4": {
             "RebelCode\\Modular\\": "src"
         }
-    }
+    },
+    "scripts": {
+        "test": "phpunit",
+        "csfix": "php-cs-fixer fix -vvv"
+    },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/Dhii/machine-loop-machine"
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/Dhii/factory-interface"
```

Comment:
This package is available [here](https://packagist.org/packages/dhii/factory-interface).